src/Tribe/Assets.php

```diff
@@ -601,17 +599,55 @@ public function remove( $slug ) {
 	 * Get the Asset Object configuration.
 	 *
 	 * @since 4.3
+	 * @since 4.11.0  Added $sort param.
 	 *
-	 * @param string $slug Slug of the Asset.
+	 * @param string|array $slug Slug of the Asset.
+	 * @param boolean      $sort  If we should do any sorting before returning.
 	 *
 	 * @return array|object|null Array of asset objects, single asset object, or null if looking for a single asset but
 	 *                           it was not in the array of objects.
 	 */
-	public function get( $slug = null ) {
+	public function get( $slug = null, $sort = true ) {
 		if ( is_null( $slug ) ) {
+			if ( $sort ) {
+				$cache_key_count = __METHOD__ . ':count';
+				// Sorts by priority.
+				$cache_count = tribe_get_var( $cache_key_count, 0 );
+				$count       = count( $this->assets );
+
+				if ( $count !== $cache_count ) {
+					uasort( $this->assets, 'tribe_sort_by_priority' );
+					tribe_set_var( $cache_key_count, $count );
+				}
+			}
 			return $this->assets;
 		}
 
+		// If slug is an array we return all of those.
+		if ( is_array( $slug ) ) {
+			$assets = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 272fd43a38ce1207dea3516dbd96134c</sub>